### DISCUSSION TITLE
java

### DISCUSSION THREAD

**Discussion by Namya48** [community/community#20500] [Other Feature Feedback, Questions, & Ideas] [answered]
hi if i want to compare numbers using all in  one then how to do it.
eg : i want to compare numbers user who is running this code will answer as "--- is greater than ---" or "--- is lesser than ---".
user will just type two numbers and he/she will get answer that the number is greater or lesser

**Comment by rilysh** [comment]
If I understood the question correctly, so the concept will be like this
```java
public class MyClass {
    public static void main(String args[]) {
        int firstNumber = 10;
        int secondNumber = 20;
        
        if (firstNumber > secondNumber) {
            System.out.println(firstNumber + " is greater than " + secondNumber);
        } else {
            System.out.println(firstNumber + " is lower than " + secondNumber);
        }
    }
}
```
For example here, if `firstNumber` integer is greater than `secondNumber` integer, it will show `firstNumber + " is greater than " + secondNumber` if not (else) it will show `firstNumber` integer is lower than `secondNumber`. You can change it like vice versa.
You can implement first number and second number as an user input, for example: `java.util.Scanner` class.

**Reply by Namya48** [reply]
you r patialy right 
actually i want that the user will write the number and he/she will get the output as great or less or equal
thank you for your rply
i have got the output for that

**Comment by chikeb-aymen** [comment, marked as answer]
Is too easy to do this (Basic of java)

  ```
import java.util.Scanner;
  
  public class Main {
      public static void main(String[] args) {
          Scanner sc = new Scanner(System.in);
          int firstNumber = sc.nextInt();
          int secondNumber = sc.nextInt();
  
          System.out.println("Enter Two Numbers " + "(Press Enter after each):");
  
          if (firstNumber > secondNumber) {
              System.out.println(firstNumber + " is greater than " + secondNumber);
          } else {
              System.out.println(firstNumber + " is lower than " + secondNumber);
          }
      }
  }
```

**Reply by Namya48** [reply]
thanks for your code 
but there is one prblm in that we have to first enter our numbers, you can see properly when yiu run your code in vs code
